src/scripts.js

```diff
@@ -52,22 +49,23 @@ function generateUser() {
 
 // CREATE RECIPE CARDS
 function createCards() {
-  fetch('http://localhost:3001/api/v1/recipes')
+  return fetch('http://localhost:3001/api/v1/recipes')
```

Comment:
Wonder why it was necessary to `return` the `fetch` -- didn't know that was a thing!

Reply:
I ended up returning it so that i could do a .then() in the page load function (line 29 of scripts.js). Otherwise the findTags() function would execute before the API was done fetching. It isn't really necessary at this point since the recipe data is still being populated by the data file but I realized that was going to be an issue in the future!